modules/RTC/JitsiRemoteTrack.js

```diff
@@ -318,6 +392,27 @@ export default class JitsiRemoteTrack extends JitsiTrack {
         return `readyState: ${readyState}, muted: ${muted}, enabled: ${enabled}`;
     }
 
+    /**
+     * Updates track's connection status.
+     * @param {string} state the current track connection state.
+     * {@link TrackStreamingStatus}.
+     * @private
+     */
+    _setConnectionStatus(status) {
+        this._connectionStatus = status;
+    }
+
+    /**
+     * Return track's connectivity status.
+     *
+     * @returns {string} the connection status
+     * <tt>TrackStreamingStatus</tt> of the track.
+     * {@link TrackStreamingStatus}.
+     */
+    getConnectionStatus() {
```

Comment:
I'll update the naming throughout the code. I'll thinking about the no listener scenario a bit more

Reply:
I think when there are no listener registered it makes more sense to be set to `null` since `_trackStreamingStatusImpl` hasn't been initialized yet to handle the streaming status. I'm not sure where else this would be used but it might lead to some confusion if this always active